dtfs-central-api/package.json

```diff
@@ -37,6 +37,7 @@
     "express-mongo-sanitize": "^2.2.0",
     "express-rate-limit": "^6.11.2",
     "express-validator": "^7.0.1",
+    "lodash.sortby": "^4.7.0",
```

Comment:
Lodash actually discourage per method packages like this as they will be removed in future versions (full reasoning [here](https://lodash.com/per-method-packages)), so I think we we should just add the whole package here.

This won't necessarily increase the build size as we can still use individual functions where required. e.g.
```
const sortBy = require('lodash/sortBy');
```
which means only the `sortBy` function will appear in the build output.

This also means we can more easily use further lodash functions in future without the need to keep adding individual packages.

Reply:
Ah, good to know, thanks!